packages/transaction-controller/src/utils.ts

```diff
@@ -60,38 +62,65 @@ export function validateTxParams(
   txParams: TransactionParams,
   isEIP1559Compatible = true,
 ) {
-  if (
-    !txParams.from ||
-    typeof txParams.from !== 'string' ||
-    !isValidHexAddress(txParams.from)
-  ) {
-    throw rpcErrors.invalidParams(
-      `Invalid "from" address: ${txParams.from} must be a valid string.`,
-    );
-  }
+  validateEIP1559Compatibility(txParams, isEIP1559Compatible);
+  validateFrom(txParams);
+  validateRecipient(txParams);
+  validateInputValue(txParams.value);
```

Comment:
The transaction context is implied, but should we rename all these functions to `validateParamX` for clarity?